### ISSUE TITLE
Sample data in home folder

### ISSUE THREAD

**Issue by egyedia** [metadatacenter/cedar-project#378]
Once a user registers, and logs in the first time, he faces an empty workspace.

It would be nice to have some sample data there.
We could have:
 - two sample elements
 - one template just with fields
 - one templates using the two elements
 - one templates with multiple instance fields
 - one template with multiple instance elements
 - some instances of the above

Or we could even have these in a "Sample data" folder

How we could do this:
  The above would be present in a specific folder of the cedar-admin user.
  Once the user logs in the first time, we would copy the content of that given folder to the user's folder. This way he/she could throw it away, if it is not needed any more.

**Comment by graybeal** [comment]
Really like this idea. Suggest this be done in a "Tutorial" folder, and that (somehow!) the folder be linked to tutorial materials.

Oh! OK, here's how. The "Read Me First" metadata instance uses the HTML field, video field, etc. to include descriptions to the user about each artifact. (It has been originally created from a Read Me First master template somewhere not in their directory, or copied in their directory if we want them to "look at the source".) They get to read through the file, filling out some metadata as they go. Then at the end they are directed to an on-line tutorial for working with the other content in the folder.

This little problem suggests that a "Note" or "Document" content type might be quite useful, for real users as well. 

Note: When you copy a folder, what happens to the IDs of templates and instances in the new folder? Some instances may point back to original folder's templates, no?

**Comment by martinjoconnor** [comment]
Good idea. Also like the self-documenting templates. Putting in 1.2 because 1.1 is a little crowded.

**Comment by graybeal** [comment]
I used this as a strategy for presenting talk bullets within a CEDAR demo.  Worked pretty well. Except I couldn't link to another URL in CEDAR, oddly.
 
(https://cedar.staging.metadatacenter.net/templates/edit/https://repo.staging.metadatacenter.net/templates/e45b5e04-e06e-46da-92e3-2ed89c676545?folderId=https:%2F%2Frepo.staging.metadatacenter.net%2Ffolders%2Fb6cb3d33-ad8b-4a2e-bbdb-a237f02cf3d4).

**Comment by martinjoconnor** [comment]
Pretty cool. We should definitely put that in the initial set.

**Comment by willrett** [comment]
Fancy
